crates/compilers/Cargo.toml

```diff
@@ -66,9 +67,12 @@ reqwest = "0.12"
 tempfile = "3.9"
 snapbox.workspace = true
 foundry-compilers-core = { workspace = true, features = ["test-utils"] }
+rstest = "0.24.0"
+flate2 = "*"
+tar = "0.4.44"
 
 [features]
-default = ["rustls"]
+default = ["rustls", "async", "svm-solc", "project-util"]
```

Comment:
why do we need these in `default` features now? I see that they were used in `full`